home/.chezmoiscripts/universal/run_onchange_after_19-android-sdk.sh.tmpl

```diff
@@ -0,0 +1,30 @@
+{{- if ne .host.distro.family "windows" -}}
+#!/usr/bin/env bash
+# @file Android SDK Pre-Install
+# @brief Pre-installs a list of Android SDKs and Tools
+# @description
+#     This script pre-installs Android SDKs and Tools defined in [`.chezmoidata.yaml`](https://github.com/megabyte-labs/install.doctor/tree/master/home/.chezmoidata.yaml)
+#     so that the user does not have to do anything on launching Android Studio for the first time.
+
+{{ includeTemplate "universal/profile" }}
+{{ includeTemplate "universal/logg" }}
+
+### Find the location of `sdkmanager`
+SDKMANAGER=""
+if command -v sdkmanager > /dev/null; then
+    SDKMANAGER="sdkmanager"
+elif [ -f $HOME/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager ]; then
+    SDKMANAGER="$HOME/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager"
+elif [ -f $HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager ]; then
+    SDKMANAGER="$HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager"
+else
+    logg info '`sdkmanager` is not installed or it is not available in PATH'
+fi
+
+### Run `sdkmanager`, if it is installed, to install SDKs and Tools
+if [ ${#SDKMANAGER} -gt 0 ]; then echo 'yes';fi
```

Comment:
Looks like this is exiting early. Fix the if then fi fi

Reply:
I left some debug code in by mistake. Fixed.